strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/nuget/NugetSearchRequest.java

```diff
@@ -4,9 +4,13 @@
 {
 
     private String filter;
+
     private String searchTerm;
+
     private String targetFramework;
 
+    private boolean includePreRelease;
```

Comment:
@anki2189 can you please make this parameter as `Boolean`?

Reply:
Fixed.